src/main/java/org/apache/sysml/api/mlcontext/MLContextConversionUtil.java

```diff
@@ -187,6 +189,38 @@ public static MatrixObject matrixBlockToMatrixObject(String variableName, Matrix
 	}
 
 	/**
+	 * Convert a {@code FrameBlock} to a {@code FrameObject}.
+	 * 
+	 * @param variableName
+	 *            name of the variable associated with the frame
+	 * @param frameBlock
+	 *            frame as a FrameBlock
+	 * @param matrixMetadata
+	 *            the matrix metadata
+	 * @return the {@code FrameBlock} converted to a {@code FrameObject}
+	 */
+	public static FrameObject frameBlockToframeObject(String variableName, FrameBlock frameBlock,
+			MatrixMetadata matrixMetadata) {
+		try {
+			MatrixCharacteristics matrixCharacteristics;
+			if (matrixMetadata != null) {
+				matrixCharacteristics = matrixMetadata.asMatrixCharacteristics();
+			} else {
+				matrixCharacteristics = new MatrixCharacteristics();
+			}
+			MatrixFormatMetaData mtd = new MatrixFormatMetaData(matrixCharacteristics,
+					OutputInfo.BinaryBlockOutputInfo, InputInfo.BinaryBlockInputInfo);
+			FrameObject frameObject = new FrameObject(MLContextUtil.scratchSpace() + "/"
+					+ variableName, mtd);
+			frameObject.acquireModify(frameBlock);
+			frameObject.release();
+			return frameObject;
+		} catch (CacheException e) {
+			throw new MLContextException("Exception converting MatrixBlock to MatrixObject", e);
```

Comment:
Please update line 219 MatrixBlock to FrameBlock and MatrixObject to FrameObject.